### PR TITLE
Phase 1 training for BoostedDoubleSV tagger for AK8 jets

### PR DESCRIPTION
**Pull request by ferencek**
New Phase 1 training for BoostedDoubleSV tagger for AK8 jets.

Request to add new training to externals is here cms-sw/cmsdist#2951

@cvernier

### CODE REVIEW

**Comment by ferencek** [comment]
Please hold off any tests until the external training file is integrated into 91X IBs.

**Comment by iahmad-khan** [comment]
please test with cms-sw/cmsdist#2953

File: RecoBTag/SecondaryVertex/python/candidateBoostedDoubleSecondaryVertexAK8Computer_cfi.py [hunk 1]
```diff
@@ -8,3 +8,6 @@
     useGBRForest = cms.bool(True),
     useAdaBoost = cms.bool(False)
 )
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(candidateBoostedDoubleSecondaryVertexAK8Computer, weightFile = cms.FileInPath('RecoBTag/SecondaryVertex/data/BoostedDoubleSV_AK8_BDT_PhaseI_v1.weights.xml.gz'))
```
Comment (slava77) [inline RecoBTag/SecondaryVertex/python/candidateBoostedDoubleSecondaryVertexAK8Computer_cfi.py]: what is the plan to have this in GT?

Comment (ferencek) [inline RecoBTag/SecondaryVertex/python/candidateBoostedDoubleSecondaryVertexAK8Computer_cfi.py]: At the moment there is no plan. From the past experience it is much easier, more convenient, and faster to handle training files as externals. Perhaps things are better now with GTs but I doubt we can get a new GT created on the same times scale on which externals can be integrated in the IBs.

**Comment by slava77** [comment]
@ferencek 
what is the plan to add these taggers to DQM?

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/18318	7f2561c
- changes are as described
- jenkins tests pass and comparisons with baseline show no differences (the double-boosted tagger is not monitored)

**Comment by slava77** [comment]
@davidlange6 
just FYI, the external needed for this PR was merged to ```cms-sw:IB/CMSSW_9_1_X/gcc530``` on Apr 13

**Comment by ferencek** [comment]
@slava77, at the moment DQM is implemented only for taggers run in the standard reco. And in the standard reco we run b tagging only on ak4 jets. At some point in the future this tagger could be included in DQM but for now there are no concrete plans that I am aware of.

**Comment by slava77** [comment]
On 4/18/17 7:04 AM, Dinko Ferencek wrote:
> @slava77 <https://github.com/slava77>, at the moment DQM is implemented
> only for taggers run in the standard reco. And in the standard reco we
> run b tagging only on ak4 jets. At some point in the future this tagger
> could be included in DQM but for now there are no concrete plans that I
> am aware of.
> 

double-boosted is a part of standard processing (miniAOD step).
So, it should be monitored in the corresponding miniAOD DQM/validation
sequences.

**Comment by ferencek** [comment]
Thanks for pointing out the MiniAOD-specific DQM sequence. To be honest, it completely slipped my mind. We will then have to add this to the development plans.
@mverzett @JyothsnaKomaragiri @imarches @cvernier

**Pull request merged**